### ISSUE TITLE
QT raises a access denied error while setting monitor DPI

### ISSUE THREAD

**Issue by FirefoxMetzger** [pygfx/pygfx#368]
While running the hello-world example from the guide on Windows 10 I've encountered the following warning/error:

```
qt.qpa.windows: setProcessDpiAwarenessContext(DPI_AWARENESS_CONTEXT_PER_MONITOR_AWARE_V2) failed: COM error 0x5  (Access is denied.)
```

Has one of you encountered this before and knows the origin? I've tried to google it to see if it is specific to my system/config, but no luck. Tests are passing and the examples render fine. The one thing that doesn't work is the example from the guide (currently only displaying a black screen).

This morning, @almarklein told me that you have to manually install a rendering backend (which I didn't do while setting up pygfx). My guess is that I might be having an outdated version of QT lying around which is getting picked up. Are there instructions on how to install the different backends that wgpu can use?

**Comment by berendkleinhaneveld** [comment]
Are you using 6.3? @ivoflipse didn't you get this message as well a few weeks ago?
You can try an older version of Qt/Pyside instead and see if that will display something other than black. Or maybe try 6.4?

**Comment by FirefoxMetzger** [comment]
Ah, that's where I got Qt from ... since the examples depend on pyside I ended up installing a backen while getting the dependencies for the examples. Thanks @berendkleinhaneveld !

> Are you using 6.3? 

No, I was actually using 6.4 . Downgrading to 6.3 removes the warning/error log.

I still only get a black screen, but I suspect that this is because the guide is outdated. All examples I've tried so far render just fine.

**Comment by Korijn** [comment]
I'm a little bummed that PySide6 continues to break things every minor update. I expect this will continue to be a source of issues.

**Comment by almarklein** [comment]
Same error for me on Win11 with PySide 6.4 indeed. Not on MacOS.

> I'm a little bummed that PySide6 continues to break things every minor update.

Sigh ... I just found out that in 6.4 enums are only available at the class, not at the instance. Good luck updating your codebase to that and not missing anything ...

**Comment by ivoflipse** [comment]
Correct, I got the following errors when running one of our projects on Windows 10 with PySide 6.3.2:

```
qt.qpa.windows: setProcessDpiAwarenessContext(DPI_AWARENESS_CONTEXT_PER_MONITOR_AWARE_V2) failed: COM error 0x5  (Access is denied.)
QWindowsWindow::setGeometry: Unable to set geometry 175x186+0+0 (frame: 175x186+0+0) on QWidgetWindow/"wgpuwidgetClassWindow" on "\\.\DISPLAY2". Resulting geometry: 175x186+2592+362 (frame: 175x186+2592+362) margins: 0, 0, 0, 0)
QWindowsWindow::setGeometry: Unable to set geometry 173x184+1+1 (frame: 173x184+1+1) on QWidgetWindow/"wgpuwidgetClassWindow" on "\\.\DISPLAY2". Resulting geometry: 173x184+2593+554 (frame: 173x184+2593+554) margins: 0, 0, 0, 0)
QWindowsWindow::setGeometry: Unable to set geometry 144x184+1+1 (frame: 144x184+1+1) on QWidgetWindow/"wgpuwidgetClassWindow" on "\\.\DISPLAY2". Resulting geometry: 144x184+2564+554 (frame: 144x184+2564+554) margins: 0, 0, 0, 0)
QWindowsWindow::setGeometry: Unable to set geometry 146x186+0+0 (frame: 146x186+0+0) on QWidgetWindow/"wgpuwidgetClassWindow" on "\\.\DISPLAY2". Resulting geometry: 146x186+2563+362 (frame: 146x186+2563+362) margins: 0, 0, 0, 0)
```
